packages/salesforcedx-vscode-core/src/commands/forceStartApexDebugLogging.ts

```diff
@@ -186,7 +184,7 @@ export class UpdateTraceFlagsExecutor extends SfdxCommandletExecutor<{}> {
     return new SfdxCommandBuilder()
       .withArg('force:data:record:update')
       .withFlag('--sobjecttype', 'TraceFlag')
-      .withFlag('--sobjectid', developerLogTraceFlag.getTraceFlagId())
+      .withFlag('--sobjectid', developerLogTraceFlag.getTraceFlagId()!)
```

Comment:
Could you comment on why the `!` is safe here?
I like to keep track of when we explicitly cast something that can be null|undefined.

Reply:
The traceflagId should be set in the execute method prior to calling this executor when running the query executor. If the field is not present in the JSON  that it's parsing, then it should catch the error and surface a failure to the user.